python/ray/train/lightning/lightning_utils.py

```diff
@@ -150,6 +184,35 @@ def teardown(self):
         pass
 
 
+@PublicAPI(stability="alpha")
+def prepare_trainer(trainer: pl.Trainer) -> pl.Trainer:
+    """Prepare the PyTorch Lightning Trainer for distributed execution."""
+
+    # Check strategy class
+    valid_strategy_class = [RayDDPStrategy, RayFSDPStrategy, RayDeepSpeedStrategy]
+
+    if not any(isinstance(trainer.strategy, cls) for cls in valid_strategy_class):
+        raise RuntimeError(
+            f"Invalid strategy class: {type(trainer.strategy)}. To use "
+            "PyTorch Lightning with Ray, the strategy object should be one of "
+            "[RayDDPStrategy, RayFSDPStrategy, RayDeepspeedStrategy] class "
```

Comment:
```suggestion
            "[RayDDPStrategy, RayFSDPStrategy, RayDeepSpeedStrategy] class "
```

or something like this?
```suggestion
            f"{[cls.__name__ for cls in valid_strategy_class]} class "
```